README.md

```diff
@@ -203,6 +203,16 @@ Landing page is static and different CSS and JS due to time constraints.
 
 This copies over necessary artifacts to `resources` dir.
 
+### Seed postgres for local development
+
+Postgres can be seeded with initial values to make sure that your local SOB app loads with some data. Just
+include the number of entities you wish to create as the first argument to `seed-postgres`, such as:
+
+```
+lein run -m commiteth.db.seed/seed-postgres 3
+```
+
+*Note, it may take a minute for all of your newly created entities to become visible in the local app.
```

Comment:
Is this because of reloading? It seems a bit arbitrary without more context. Maybe we can say:

> Note that the app reloads bounties periodically, if you want to see the newly created bounties immediately, just refresh the page :)

Reply:
Please add a link to this to the `PostgreSQL` section of the README.

Reply:
yikes, yea that is an arbitrary explanation. I will look into the exact cause and try and be more specific :)

and 👍 on adding to the `PostgreSQL` part of the readme.